src/hwProvider.spec.ts

```diff
@@ -233,21 +233,24 @@ describe("test hwProvider", () => {
     });
 
     it("should signMessage", async () => {
-        const message = new SignableMessage({
-            message: Buffer.from("Hello World"),
+        const messageToSign = new Message({
+            data: Buffer.from("Hello World"),
             address: Address.fromBech32("erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th"),
             version: 42
-        });
+        }); 
+
 
         hwApp.messageSignature = "abba";
 
-        const signedMessage = await hwProvider.signMessage(message);
+        const messageComputer = new MessageComputer();
+
+        const signedMessage = await hwProvider.signMessage(messageToSign);
 
-        assert.equal(signedMessage.message.toString(), "Hello World");
-        assert.equal(signedMessage.address.toString(), "erd1qyu5wthldzr8wx5c9ucg8kjagg0jfs53s8nr3zpz3hypefsdd8ssycr6th");
+        assert.equal(Buffer.from(signedMessage.data).toString(), "Hello World");
```

Comment:
The test is somehow legacy (does not bring very much value).

Maybe the preparation of the data to be signed (get message size, concatenation) can be extracted in a function, and unit tested. Optional.